### PR TITLE
Fix TG_OWT_USE_PIPEWIRE

### PR DESCRIPTION
**Pull request by ilya-fedin**
It seems it was never working properly and no one noticed that

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -2257,6 +2258,15 @@ if (NOT TG_OWT_USE_PROTOBUF)
     )
 endif()
 
+if (NOT TG_OWT_USE_PIPEWIRE)
+    remove_target_sources(tg_owt ${webrtc_loc}
+        modules/desktop_capture/linux/base_capturer_pipewire.cc
+        modules/desktop_capture/linux/base_capturer_pipewire.h
+        modules/desktop_capture/linux/egl_dmabuf.cc
+        modules/desktop_capture/linux/egl_dmabuf.h
```
Comment (klemensn) [inline CMakeLists.txt:2266]: Avoiding the build of `modules/desktop_capture/linux/egl_dmabuf.cc` is exactly what I had to do to fix building on OpenBSD today.

File: CMakeLists.txt [hunk 2]
```diff
@@ -121,6 +121,8 @@ if (TG_OWT_USE_PIPEWIRE)
         link_pipewire(tg_owt)
     endif()
 
+    link_glib(tg_owt)
+
```
Comment (klemensn) [inline CMakeLists.txt:125]: How is linking glib related to building with(out) pipewire?

**Comment by klemensn** [comment]
>  and no one noticed that

That's not accurate, though, is it?
You are linking against my issue that says "Fix TG_OWT_USE_PIPEWIRE=OFF build" so clearly I've "noticed that".

File: CMakeLists.txt (hunk 2 above)
Comment (ilya-fedin) [reply CMakeLists.txt:125]: glib is used only by pipewire-related code for dbus communication

**Comment by ilya-fedin** [comment]
> You are linking against my issue that says "Fix TG_OWT_USE_PIPEWIRE=OFF build" so clearly I've "noticed that".

Well, I saw your PR after I pushed my one

File: CMakeLists.txt (hunk 2 above)
Comment (klemensn) [reply CMakeLists.txt:125]: Ah, it moved into the `if (TG_OWT_USE_PIPEWIRE)` block, I missed that.

**Pull request merged**